dashboard/pages/history.snapshots.html

```diff
@@ -0,0 +1,47 @@
+{TEMPLATE:HTML_HEADER}
+
+<div class="row">
+    <div class="col-lg-12">
```

Comment:
eventuell aufteilen in:
<div class="panel-heading">
<div class="panel-body">
<div class="panel-footer">

Oben Überschrift, Mitte Bild und unten Buttons